docs/csharp/language-reference/keywords/implicit-numeric-conversions-table.md

```diff
@@ -18,24 +18,26 @@ The following table shows the predefined implicit conversions between .NET numer
 |----------|--------|  
 |[sbyte](sbyte.md)|`short`, `int`, `long`, `float`, `double`, or `decimal`|  
 |[byte](byte.md)|`short`, `ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
+|[char](char.md)|`ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
 |[short](short.md)|`int`, `long`, `float`, `double`, or `decimal`|  
 |[ushort](ushort.md)|`int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
 |[int](int.md)|`long`, `float`, `double`, or `decimal`|  
 |[uint](uint.md)|`long`, `ulong`, `float`, `double`, or `decimal`|  
 |[long](long.md)|`float`, `double`, or `decimal`|  
-|[char](char.md)|`ushort`, `int`, `uint`, `long`, `ulong`, `float`, `double`, or `decimal`|  
-|[float](float.md)|`double`|  
 |[ulong](ulong.md)|`float`, `double`, or `decimal`|  
+|[float](float.md)|`double`|  
   
 ## Remarks  
 
 - Any [integral type](integral-types-table.md) is implicitly convertible to any [floating-point type](floating-point-types-table.md).
 
 - Precision but not magnitude might be lost in the conversions from `int`, `uint`, `long`, or `ulong` to `float` and from `long` or `ulong` to `double`.  
   
-- There are no implicit conversions to the `char` type.  
+- There are no implicit conversions to the `bool`, `char`, `byte` and `sbyte` types.  
+
+- There are no implicit conversions from the `char`, `double` and `decimal` types.
```

Comment:
If the bool type is mentioned in the previous point, it should appear here as well; to be consistent.

Reply:
'bool' is removed.